src/test/java/com/tsmms/skoop/communityuser/registration/command/CommunityUserRegistrationCommandControllerTests.java

```diff
@@ -273,9 +269,8 @@ void communityMemberInvitesUsersToJoinOpenCommunity() throws Exception {
 				.userName("tester")
 				.build();
 
-		given(securityService.isAuthenticatedUserId("1f37fb2a-b4d0-4119-9113-4677beb20ae2")).willReturn(true);
-		given(securityService.isCommunityMember("123")).willReturn(true);
-		given(securityService.isCommunityManager("123")).willReturn(false);
+		given(communityQueryService.isCommunityMember(tester.getId(), "123")).willReturn(true);
+		given(communityQueryService.hasCommunityManagerRole(tester.getId(), "123")).willReturn(false);
```

Comment:
There seem to be the two methods `isCommunityMember(userId, communityId)` and `hasCommunityManagerRole(userId, communityId)` which are very similar but have a different naming pattern. Maybe it could be changed to a consistent wording, e.g. `isCommunityManager(userId, communityId)`.

Reply:
Good point. Thank you 👍 . I thought about renaming of `hasCommunityManagerRole(userId, communityId)` but left it as it had been.
Taking in account your suggestion I have just renamed `hasCommunityManagerRole(userId, communityId)` to `isCommunityManager(userId, communityId)`.